src/js/components/NodesGridView.js

```diff
@@ -65,7 +65,11 @@ var NodesGridView = React.createClass({
    * @param  {Object} props
    */
   componentWillReceiveProps: function (props) {
-    let ids = _.pluck(props.services, 'id');
+    let ids = props.services.reduce(function (acc, value) {
```

Comment:
good idea, also I can probably use map instead of pluck for a number of times which I will investigate right now